src/test/scala/is/hail/variant/GenomeReferenceSuite.scala

```diff
@@ -126,4 +126,23 @@ class GenomeReferenceSuite extends SparkSuite {
     kt5.typeCheck()
     assert(kt5.forall("v37a.isAutosomal() && !v37na.isAutosomal() && v38a.isAutosomal() && !v38na.isAutosomal()"))
   }
+
+  @Test def testConstructors() {
+    val kt = hc.importTable("src/test/resources/sampleAnnotations.tsv")
+    val ktann = kt.annotate("""v1 = Variant(GRCh38)("chrX:156030895:A:T"), v2 = Variant(GRCh37)("X:154931044:A:T"),
+    |v3 = Variant(GRCh37)("1", 3, "A", "T"), l1 = Locus(GRCh38)("1", 100), l2 = Locus(GRCh37)("1:100"),
+    |i1 = Interval(GRCh37)("1:5-10"), i2 = Interval(GRCh38)("chrX", 156030890, 156030895)""".stripMargin)
```

Comment:
If I create a Variant/Interval/Locus without a reference what type does it get? Let's add a test for that.